src/platform/forms-system/src/js/utilities/file/PdfPasswordFeature.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+
+// Feature toggle - made a class component because only a class can be used
+// with the `withRef` connect option
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
Because I didn't want to use `connect` in the FileField component, which I did to avoid adding a Provider & store is all the unit tests - which may not be necessary anymore, now that I've ended up doing it anyway - I added this `PdfPasswordFeature` component to get the feature toggle state using `connect`. The component includes a property with the set state.

When the `withRef` is set (see `react-redux` v5.x [`connectAdvanced`](https://react-redux.js.org/5.x/api/connect-advanced#arguments) docs - and honestly, I don't know why `connectAdvanced` showed an error, but a regular `connect` worked) it exposes the component within the `connect` so we can use a reference to target it.

Within the `FileField`, we have to use `getWrappedInstance()` to get access to the wrapped component in order to get the feature flag state.

Honestly, now that I look over this mess, I think I'll refactor this to have the `connect` within the `FileField` and ping you once that's done.